.github/workflows/ci.yaml

```diff
@@ -12,7 +12,12 @@ env:
   OPTIMISTIC_ETHERSCAN_API_KEY: ${{ secrets.OPTIMISTIC_ETHERSCAN_API_KEY }}
```

Comment:
*Arbitrary comment location*

Overall looking great @garyghayrat and simulation worked locally. Nice job! One nit: can we add some notes to the contracts-periphery/README file about how to use the script? Thanks!

Reply:
Thanks! Update `README` here [@netlify
Header rules - jolly-shaw-20fe62 Completed in 2m — No header rules processed
Required
[Details](https://github.com/ScopeLift/umbra-protocol/pull/504/checks?check_run_id=13327010322)](https://github.com/ScopeLift/umbra-protocol/pull/504/commits/a22e1d7c7ac9757fabeff71f5a1cf36e4b885ad7)